lemur/factory.py

```diff
@@ -162,11 +180,15 @@ def configure_extensions(app):
 
     if app.config["CORS"]:
         app.config["CORS_HEADERS"] = "Content-Type"
+        origins = app.config.get("CORS_ORIGIN", "*")
```

Comment:
Can we use stronger types here and avoid the deserialization? e.g. take in a List[str]